tests/runner/0x565556789012345678901234567890123456abcd/init_output.json

```diff
@@ -0,0 +1,16 @@
+{
+  "errors": [
+    {
+      "error_message":
+        "Cyclic dependence found when importing 0x565556789012345678901234567890123456abcd.",
```

Comment:
This error message is going to be a pain to deal with for the developer.

Symbolic names need to be a top priority for high-level extensions.

Reply:
Changing this to report the library mentioned in the contract itself is not difficult. I didn't do so assuming that addresses may be easier to point out (uniqueness).
Also note that the error location shown here corresponds to the location of the name that was mentioned in the source, so the user can easily get to it already.

However, if you think that having the name itself printed here is better, I can do that. It's simple.

Reply:
I'm now printing both. Have a look.